src/app/acesso/cadastro/cadastro-cliente/cadastro-cliente.component.ts

```diff
@@ -9,6 +9,7 @@ import {MyErrorStateMatcher} from '../../../classes/my-error-state-matcher';
 import { ConsultaEstadosService, Estado, Cidade } from 'src/app/servicos/consulta-estados.service';
 import { LoginService } from 'src/app/servicos/login.service';
 import { Validacoes } from 'src/app/validacoes';
+import { telefoneFormControl, nomeFormControl, emailFormControl, senhaFormControl, cpfFormControl } from 'src/app/util/form-controls';
```

Comment:
Codacy found an issue: [`src/app/util/form-controls` import should occur before import of `../../../../../node_modules/@angular/forms`](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6334730)